playbooks/adhoc/uninstall.yml

```diff
@@ -140,10 +140,11 @@
 
     - file: path={{ item }} state=absent
```

Comment:
Not related to this PR, but, we don't need recurse=yes to nuke trees like /etc/origin and /var/lib/origin?